rust/src/graph/mod.rs

```diff
@@ -649,7 +646,7 @@ struct PyImportDetails {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, new)]
-struct PackageDependency {
+struct PyPackageDependency {
```

Comment:
Hmm, I've just noticed there is some inconsistency in the way we prefix structs that have a Python / Rust counterpart. In the file system classes, the structs that are exposed to Python have the `Py` prefix (just like the pyo3 classes like PySet, PyDict etc.) But in this module it's the other way around - `PyImportDetails`is the pure-Rust one and `ImportDetails` is the Python one. I think that's confusing...

So I think it would be better to call this one `PackageDependency` and the other one `PyPackageDependency` (and, I guess, swap over `ImportDetails` / `PyImportDetails` too). What do you think?